src/app/shared/subscription-manager/subscription.manager.ts

```diff
@@ -1,9 +1,10 @@
 /** SubscriptionManager
 /*  stores references to subscriptions / unsubscribes from them on destroy
 */
-import { OnDestroy } from '@angular/core';
+import { Directive, OnDestroy } from '@angular/core';
 import { Subscription } from 'rxjs';
 
+@Directive()
```

Comment:
Doesn't a directive need a selector so that it can be referenced in the HTML/DOM?

Reply:
This is a superclass and only child classes are referenced by the DOM, so it would be redundant.  The decorator itself is needed in Angular 10 however - it won't compile without it.